src/Microsoft.AspNet.Mvc.Core/Description/DefaultApiDescriptionProvider.cs

```diff
@@ -414,6 +421,34 @@ private class ApiParameterContext
             public IReadOnlyList<TemplatePart> RouteParameters { get; }
         }
 
+        private class ApiDescriptorBindingContext
```

Comment:
Better name here like `ApiParameterDescriptionContext` - there is no such thing as  `ApiDescriptor`